app/jobs/regular/process_post.rb

```diff
@@ -60,6 +60,27 @@ def extract_links(post)
       TopicLink.extract_from(post)
       QuotedPost.extract_from(post)
     end
+
+    def auto_tag(post)
+      word_watcher = WordWatcher.new(post.raw)
+
+      old_tags = post.topic.tags.pluck(:name).to_set
+      new_tags = old_tags.dup
+
+      WordWatcher.words_for_action(:tag).each do |word, tags|
+        if word_watcher.word_matches?(word)
+          new_tags += tags.split(",")
+        end
```

Comment:
Could be a one-liner 👍 

```suggestion
        new_tags += tags.split(",") if word_watcher.word_matches?(word)
```

Reply:
It was like that before, but it looked a bit too long. I'll do it like this again.